lib/client/src/client/client.ts

```diff
@@ -112,6 +123,54 @@ export interface AuthInfo {
  * @template R The type to use for ranges (such as `vscode.Range` for VS Code extensions).
  */
 export interface Client<R extends Range> {
+    /**
+     * Get the capabilities returned by the configured providers.
+     *
+     * It does not continue to listen for changes, as {@link Client.capabilitiesChanges} does. Using
+     * {@link Client.capabilities} is simpler and does not require use of observables (with a library like
+     * RxJS), but it means that the client needs to manually poll for updated item kinds if freshness is
+     * important.
+     */
+    capabilities(
```

Comment:
Not strictly related, but thoughts on renaming capabilities to meta? Capabilities was inspired by LSP but it's long and it's a confusing word, and we're calling the same thing meta elsewhere and that feels more natural.